app/components/recirculation-prefooter.js

```diff
@@ -24,12 +25,15 @@ export default Component.extend(
     wikiUrls: service(),
     fetch: service(),
     sponsoredContent: service(),
+    runtimeConfig: service(),
 
     classNames: ['recirculation-prefooter'],
     classNameBindings: ['items:has-items'],
 
     listRendered: null,
+    requestId: null,
     topArticles: null,
+    fallbackItems: null,
```

Comment:
is it used for anything else? If no, maybe the variable could be renamed to "recommendationFallbackItems"?

Reply:
nah, it's not java, lets keep names short ;) The recommendations IMO does not bring any value here